UPGRADE-1.12.md

```diff
@@ -1,5 +1,30 @@
 # UPGRADE FROM `v1.12.10` TO `v1.12.11`
 
+1. Due to a bug that was causing the removal of promotion configurations for promotions [REF](https://github.com/Sylius/Sylius/issues/15201),
+   The constructor of `Sylius\Bundle\CoreBundle\EventListener\ProductDeletionListener` has been modified as follows:
+
+   ```diff
+    public function __construct(
+        private RequestStack $requestStack,
+    +   private ProductInPromotionRuleCheckerInterface $productInPromotionRuleChecker,
+    -   ProductAwareRuleUpdaterInterface ...$ruleUpdaters,
+    )
+    ```
+   
+   Please refrain from using ProductAwareRuleUpdaterInterface.
+   Instead, use the `protectFromRemovingProductThatIsAssignedToPromotionRule` method, which has replaced the `removeProductFromPromotionRules` method.
+
+1. The constructor of `Sylius\Bundle\CoreBundle\EventListener\TaxonDeletionListener` has changed:
```

Comment:
Either this should be joined with the previous point, since it's the same bug, or explain why the constructor has changed.